nuqql/win.py

```diff
@@ -63,7 +63,8 @@ def _redraw_win(self):
         self.win.clear()
 
         # color settings on
-        curses.init_pair(1, curses.COLOR_BLUE, curses.COLOR_BLACK)
+        curses.use_default_colors()
```

Comment:
Would it be possible to move curses.use_default_colors() to the start() function in ui.py, maybe just above nuqql.config.init_win(stdscr) in line 278?

Reply:
Changed it in the last commit, is it ok like that?